Makefile

```
@@ -6,6 +6,7 @@ all: help
 
 build: docs
 	python setup.py sdist --formats=gztar
+	python setup.py bdist_wheel
```

Not saying this is wrong or insisting you change it but most of the new projects use [build](https://pypi.org/project/build/) for this. Might be worth a shot since it does the build in a venv, so it verifies that all the build-time deps are pulled in correctly. Then it uses sdist to build wheel, so you have an extra check that sdist isn't missing anything. But this obviously implies that it needs to be installed prior.

```suggestion
	python -m build
```

(with no extra args it builds both sdist & wheel into `dist/`)